modules/openapi-generator-cli/src/main/java/org/openapitools/codegen/cmd/Generate.java

```diff
@@ -70,6 +70,10 @@
             description = "folder containing the template files")
     private String templateDir;
 
+    @Option(name = {"-e", "--engine"}, title = "templating engine",
+        description = "templating engine, for now \"mustache\" and \"handlebars\" are supported")
```

Comment:
Minor suggestion: make it clear that mustache is the default and handlebar is beta only (not officially support)